weather.py

```diff
@@ -12,22 +12,33 @@ def __init__(self):
             "rain_today": "n.a",
             "current_temp": "n.a",
             "hourly_rain": "n.a",
+            "sunrise": "1",
+            "sunset": "2"
         }
 
     def get_weather_printout(self):
         self.get_data()
         temp_line = f"Current: {self.weather['current_temp']}° |  High: {self.weather['high_temp']}° |  Low: {self.weather['low_temp']}°"
         rain_divider = "Hourly Rain . . . . . . . . . . . . . ."
         rain = f"{self.weather['hourly_rain']}"
+        sunrise = self.weather["sunrise"]
+        sunset = self.weather["sunset"]
         # return f"Current: {self.weather['current_temp']}° | High: {self.weather['high_temp']}° | Low: {self.weather['low_temp']}° \nRain: {self.weather['rain_today']}%"
-        return temp_line + "\n" + rain_divider + "\n" + rain
+        return temp_line + "\n" + rain_divider + "\n" + rain + "\n" + f"Sunrise: {sunrise}"
 
     def get_data(self):
         try:
-            api = f"https://api.open-meteo.com/v1/forecast?latitude=40.6501&longitude=-73.9496&current=temperature_2m,precipitation&hourly=temperature_2m,precipitation_probability,precipitation&daily=temperature_2m_max,temperature_2m_min,precipitation_probability_max&temperature_unit=fahrenheit&wind_speed_unit=mph&precipitation_unit=inch&forecast_days=3"
+            api = f"https://api.open-meteo.com/v1/forecast?latitude=40.6501&longitude=-73.9496&current=temperature_2m,precipitation&hourly=temperature_2m,precipitation_probability,precipitation&daily=temperature_2m_max,temperature_2m_min,precipitation_probability_max&temperature_unit=fahrenheit&wind_speed_unit=mph&precipitation_unit=inch&daily=sunrise,sunset&forecast_days=3"
```

Comment:
`api = f"https://api.open-meteo.com/v1/forecast?latitude=40.6501&longitude=-73.9496&current=temperature_2m,precipitation&hourly=temperature_2m,precipitation_probability,precipitation&daily=temperature_2m_max,temperature_2m_min,precipitation_probability_max&temperature_unit=fahrenheit&wind_speed_unit=mph&precipitation_unit=inch&daily=sunrise,sunset&timezone=America%2FNew_York&forecast_days=3"`

Reply:
Added timezone to payload. Should now show the correct times for sunrise and set relative to your local machine.